src/Generator/Generators/CSharp/CSharpSources.cs

```diff
@@ -364,6 +363,55 @@ public override bool VisitClassDecl(Class @class)
 
                 if (@class.IsDynamic)
                     GenerateVTable(@class);
+            } else
```

Comment:
If we make adjustments. That's why. But If you're fine without any adjustments to the generations, I'll remove it.